tutorials/hess.ipynb

```diff
@@ -72,12 +69,64 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data_store = DataStore.from_file(\n",
-    "    \"$GAMMAPY_DATA/hess-dl3-dr1/hess-dl3-dr3-with-background.fits.gz\"\n",
-    ")\n",
-    "mask = data_store.obs_table[\"TARGET_NAME\"] == \"Crab\"\n",
-    "obs_table = data_store.obs_table[mask]\n",
-    "observations = data_store.get_observations(obs_table[\"OBS_ID\"])"
+    "config = \"\"\"\n",
+    "general:\n",
+    "    logging:\n",
+    "        level: INFO\n",
+    "    outdir: .\n",
+    "\n",
+    "observations:\n",
+    "    datastore: $GAMMAPY_DATA/hess-dl3-dr1/hess-dl3-dr3-with-background.fits.gz\n",
+    "    filters:\n",
+    "        - filter_type: par_value\n",
+    "          value_param: Crab\n",
+    "          variable: TARGET_NAME\n",
+    "\n",
+    "reduction:\n",
+    "    dataset-type: MapDataset\n",
+    "    stack-datasets: true\n",
+    "    offset-max: 2.5 deg\n",
+    "    psf-kernel-radius: 0.3 deg\n",
```

Comment:
I see e.g. `psf-kernel-radius` doesn't appear here: https://docs.gammapy.org/dev/scripts/index.html
Can we get a dump of the fixed part of the schema so that this at least shows up in one place in the docs?
Also: this is a very low-level parameter, should it even exist? Shouldn't this be chosen by Gammapy from the PSF size to have good containment?
If you expose it or put any Angle as default, you have created a simple way to silently get bad analysis results and publish wrong science results, e.g. pair halos for AGN or Crab extension ...

Reply:
Yes. In particular, the value given here is too small to get correct results. We should not provide a tutorial suggesting wrong parameters to users.

Reply:
I agree we should remove this parameter for now, because it's really low level and instead choose a reasonable default.

Reply:
Done